deps/rabbit/src/rabbit_quorum_queue.erl

```diff
@@ -111,6 +115,30 @@
 -define(ADD_MEMBER_TIMEOUT, 5000).
 -define(SNAPSHOT_INTERVAL, 8192). %% the ra default is 4096
 
+%%----------- QQ policies ---------------------------------------------------
+
+-rabbit_boot_step(
+   {?MODULE,
+    [{description, "QQ policy validation"},
+     {mfa, {rabbit_registry, register,
+            [policy_validator, <<"target-qq-replica-count">>, ?MODULE]}},
```

Comment:
We can use just "target_replica_count". It will be applicable to streams, too.

Reply:
Given we have used groups-size in other config keys should it be `target_group_size`?

Reply:
Going with target_group_size for now then!